{
  try {
    failed(t,description);
  }
 catch (  Throwable t1) {
    errors.add(t1);
  }
}
